src/components/editMeasure/measureDetails/measureInformation/MeasureInformation.tsx

```diff
@@ -146,7 +146,12 @@ export default function MeasureInformation() {
       await handleSubmit(values),
   });
 
-  const canEdit = measure?.createdBy === userName;
+  const isOwner = measure?.createdBy === userName;
+  const canEdit =
+    measure?.createdBy === userName ||
```

Comment:
I just realized the left side of the OR can be replaced with `isOwner`.

Reply:
fixed